src/main/java/edu/rosehulman/quota/controller/IncrementQuotaController.java

```diff
@@ -56,8 +65,11 @@ public Object handle(Request request, Response response) throws Exception {
     }
 
     // See if adding one to the quota would go above the quota
-    BigInteger valuePlusOne = value.add(BigInteger.ONE);
-    if (valuePlusOne.compareTo(max) > 0) {
+    BigInteger newValue = value.add(BigInteger.ONE);
```

Comment:
Wouldn't this be double counting if `count != null`?

Reply:
Also maybe the variable name can be something like `incrementedValue` to be more descriptive